ui/src/components/SelectImportPage/screens/SelectProjectsScreen/index.tsx

```diff
@@ -78,66 +80,72 @@ export const SelectProjectsScreen = ({
-              onClick={() => setPage((prevPage) => prevPage + 1)}
-              isLoading={!!projects.length && isProjectsLoading}
-            >
-              Load More
-            </LoadingButton>
-          </CenterWrapper>
-        ) : null}
-      </>
-
-      <ButtonWrapper>
-        <Button onClick={() => handleNavigateToConnectedPage()}>Cancel</Button>
-        <LoadingButton
-          appearance='primary'
-          isDisabled={selectedProjects.length === 0}
-          onClick={() => handleNavigateToScreen()}
-          isLoading={isProjectsImporting}
-        >
-          Select
-        </LoadingButton>
-      </ButtonWrapper>
+      <TableHeaderWrapper>
+        <GroupSelectorWrapper data-testid='group-selector'>
+          <Select
+            isClearable
+            isLoading={isGroupsLoading}
+            isDisabled={isProjectsLoading && !isGroupsLoading}
+            onChange={(e) => handleChangeGroup(e)}
+            inputId='select-group'
+            className='single-select'
+            classNamePrefix='react-select'
+            placeholder='Select group'
+            options={groupSelectorOptions}
+          />
+        </GroupSelectorWrapper>
+        <TableSearchWrapper>
+          <Search handleSearchValue={handleSearchValue} isProjectsLoading={isProjectsLoading} />
+        </TableSearchWrapper>
+      </TableHeaderWrapper>
+      <ProjectsImportTable
+        projects={projects}
+        isLoading={isProjectsLoading && !projects.length}
+        onSelectAllItems={onSelectAllItems}
+        onSelectItem={onSelectItem}
+        onChangeComponentType={onChangeComponentType}
+        error={projectsFetchingError}
+        importableComponentTypes={importableComponentTypes}
+        teamsResult={teamsResult}
+        selectProjectTeam={selectProjectTeam}
+        isSpotlightActive={isSpotlightActive}
+        finishOnboarding={finishOnboarding}
+      />
+      {projects.length !== 0 ? (
+        <>
+          <Divider />
+          <Inline spread='space-between' alignBlock='center'>
+            <p>
+              <strong>{Object.keys(selectedProjects).length}</strong>{' '}
+              {projectsToImportMessage(Object.keys(selectedProjects).length)}
+            </p>
+            {totalProjects > projects.length ? (
+              <Button
+                testId={'load-more-button'}
+                label={'Load More'}
+                onClick={() => setPage((prevPage) => prevPage + 1)}
+                isDisabled={totalProjects <= projects.length}
+              >
+                {!!projects.length && isProjectsLoading ? <Spinner /> : 'Load More'}
+              </Button>
+            ) : null}
+            <ButtonWrapper>
+              <Button onClick={() => handleNavigateToConnectedPage()}>Cancel</Button>
+              <LoadingButton
+                appearance='primary'
+                isDisabled={selectedProjects.length === 0}
+                onClick={() => handleNavigateToScreen()}
```

Comment:
nit; this and the other click handler can be `onClick={handleNavigateToScreen}`

this is a good habit because it doesn't create an anonymous function on every render